swift-generator/pom.xml

```diff
@@ -93,5 +93,31 @@
       <groupId>io.airlift</groupId>
       <artifactId>log</artifactId>
     </dependency>
+    
+    <dependency>
+      <groupId>com.facebook.swift</groupId>
+      <artifactId>swift-test-utils</artifactId>
+      <scope>test</scope>
+    </dependency>
+    
+    <dependency>
+      <groupId>org.testng</groupId>
+      <artifactId>testng</artifactId>
+      <scope>test</scope>
+    </dependency>
+    
   </dependencies>
+  
+  <build>
+      <plugins>
+          <plugin>
```

Comment:
properties, meaning Maven properties? These are not the same. This is telling Surefire to set Java system properties before executing the tests - this way we can put the output directory for the generated files to be in Maven specific locations (if we moved to gradle, for example, we would just change the property, but not the tests).